omnibus/config/projects/supermarket.rb

```diff
@@ -29,8 +29,11 @@
 build_iteration 1
 
 # NOTE: see the omnibus-supermarket cookbook gemfile for controlling the infra client version
-override :postgresql, version: '9.3.25'
+override :postgresql, version: '13.3'
 override :ruby, version: "2.7.4"
+override :rubygems, version: "3.1.2" # rubygems ships its own bundler which may differ from bundler defined below and then we get double bundler which makes the omnibus environment unhappy. Make sure these versions match before bumping either.
+override :bundler, version: "2.1.2" # this must match the BUNDLED WITH in all the repo's Gemfile.locks
```

Comment:
It looks like we have some old stuff coming back in this PR

Reply:
sure, let me remove this. Thanks for pointing this out @tas50